docs/pages/desktop-access/getting-started.mdx

```diff
@@ -314,6 +314,62 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
   ![Disable Require](../../img/desktop-access/disable.png)
 </Figure>
 
+### Configure certificate for RDP connections
+
+<Admonition type="note" title="Secure Cipher Suites">
+  Teleport's RDP client supports only secure algorithms
+  for making TLS connections, so we have to configure a Windows server
+  to support those cipher suites as well.
+  This step is only *necessary* for Windows Server 2012 R2 as it does not support
+  secure algorithms by default.
```

Comment:
If we're advising some users to skip this step, I think we should add a link to the next step here.

Reply:
Great suggestion! It'll make it a bit easier for users to follow the steps in our guide.

Reply:
Is there a way to know what will be the link to the header? I see that the links are auto-generated based on the text, so I went to the current live docs and took it from there. I was wondering if there is a better way to do it?

Reply:
That's what I do too. If you wanted to be explicit you could stick in some plain HTML  (`<h2 id="myanchor">Text</h2>`), but I think the preference is for markdown.

Reply:
Btw, you can render locally using https://github.com/gravitational/next - either checkout or create your branch in the submodule within the sources.